### ISSUE TITLE
CAPTCHA query not always reliable

### ISSUE THREAD

**Issue by rgrenz** [adamzammit/limesurvey-docker#32]
Hi, thank you for sharing the image!

I noticed a small issue when using a captcha to restrict access to a survey. Sometimes, the captcha does not show up, and one has to click the reload button a few times for it to appear. 

The request made for retrieving the captcha (`/index.php/survey/captcha?v=...`) then returns the following:
```
500: Internal Server Error
imagecolorallocate(): Argument #2 ($red) must be between 0 and 255 (inclusive)
```

Not sure if this is a problem within this image or of LimeSurvey in general. Sorry if this is the wrong place to raise this issue.

**Comment by adamzammit** [comment]
This looks like an upstream bug/issue related to PHP 8: https://bugs.limesurvey.org/view.php?id=18117

I'll see if I can bump up the priority of that bug upstream

**Comment by adamzammit** [comment]
I fixed this upstream and it should now be fixed in the 5.3.26 release: https://github.com/adamzammit/limesurvey-docker/commit/5acb812dc6f58ec61d904b88d4c42ffcdeba0e96
